lib/orderbook/OrderBook.ts

```diff
@@ -131,6 +132,14 @@ class OrderBook extends EventEmitter {
     });
   }
 
+  /**
+   * Get all trades or a limited number of trades.
```

Comment:
I think it would be good to add "from the database" to the end of the description of the method.